pkg/api/annotations/annotations.go

```diff
@@ -16,8 +16,35 @@ limitations under the License.
 
 package annotations
 
-const kubectlPrefix = "kubectl.kubernetes.io/"
+const (
```

Comment:
Discussed offline. We are moving the external types to k8s.io/api, see #44784. If we make `pkg/api/annotation` the home for annotation keys, the package needs to be moved to k8s.io/api as well. Then some internal code, like the registry code, will have to import k8s.io/api, which looks weird. Hence, we decided to follow the convention to duplicate the definitions in annotation_key_constants.go.